examples/benchmarks/report.py

```diff
@@ -0,0 +1,99 @@
+import csv
+import sys
+import os
+import time
+from .CSVTableBase import CsvTableBase  #fix
+
+sys.path.append("../../")
+# sys.path.append("../../../cspy")
+from vrpy.main import VehicleRoutingProblem
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CsvTableVRP(CsvTableBase):
+    def __init__(self, *args, **kwargs):
+        #self super thingy
+        super().__init__(*args, **kwargs)
+
+    def write_to_file(self, path='report'):
+        super().write_to_file(path)
+
+    def _find_optimal(self):
+        #se på filen jeg skal parse
+        #solomon har ikke noe å parse, kanskje ta den optimale verdien som input
+        if self.instance_type == "augerat":
+            with open(self.path + self.instance_name, 'r') as fp:
+                fp.readline(),
+                beststring = (str(fp.readline())[:-2]).split()[-1]
+                self.best_known_solution = int(beststring)
+            fp.close()
+        if self.instance_type == "solomon":
+            self.best_known_solution = None
+
+        if self.instance_type == "cordeau":
+            self.best_known_solution = None
+            #sjekk om det finnes en
+
+    def get_data_from_VRP_instance(self,
+                                   prob=None,
+                                   best_known_solution=None,
+                                   subproblem_type=None,
+                                   dive=None,
+                                   pricing_strategy=None,
+                                   *args,
+                                   **kwargs):
+
+        time1 = time.time()
+        prob.solve(**kwargs, dive=dive)
+        time2 = time.time()
+
+        if best_known_solution == None:
+            self._find_optimal()
+        else:
+            self.best_known_solution = best_known_solution
+
+        self.dive = dive if not dive == None else False
+        self.pricing_strategy = pricing_strategy
+        self.subproblem_type = subproblem_type
+
+        self.comp_time = time2 - time1
+        self.upper_bound = prob.best_value
+        self.lower_bound = prob._lower_bound[-1]  #bør lower bound = lowerbound

```

Comment:
This won't be meaninngful when diving unless main.py [lines](https://github.com/Halvaros/vrpy/blob/c7679366daed319be8eea709205633f987f52d4d/vrpy/main.py#L290-L291) are changed to 
```python
duals, _ = self.masterproblem.solve_and_dive(
                time_limit=self._get_time_remaining())
```